students/rmart300/lesson09/mailroom.py

```diff
@@ -0,0 +1,36 @@
+from datetime import date
+from donor_collection import DonorCollection
+
+def validate_user_selection(action, action_dict):
+    """ validate the user selection versus available actions """
+
+    key_list = list(action_dict.keys())
+    try:
+        if int(action) not in key_list:

```

Comment:
This can be `if int(action) not in action_dict:` without the assignment to `key_list` because the `in` operator checks for key membership in dict